consensus/src/metrics_safety_rules.rs

```diff
@@ -125,3 +148,121 @@ impl TSafetyRules for MetricsSafetyRules {
         })
     }
 }
+
+pub struct MockSafetyRules {
```

Comment:
Let's move this struct into the test module (e.g., below the unit test itself) -- I don't think it needs to be exposed to non-test code 😄

Reply:
Moved this struct inside mod tests